SCClassLibrary/SCDoc/SCDocRenderer.sc

```diff
@@ -119,17 +120,34 @@ SCDocHTMLRenderer {
 			baseDir = baseDir ++ "/..";
 		};
 
+		thisIsTheMainHelpFile = (doc.title=="Help") and:
+			{((thisProcess.platform.name===\windows) and: (folder=="Help")) or: {folder==""}};
+
+		stream
+		<< "<!doctype html>"
+		<< "<html lang='en'>"
```

Comment:
np. we could probably just add it in based on directory hierarchy, but that doesn't exist right now so i guess it's fine as is